cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,15 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://conduit.mate.academy/');
```

Comment:
Avoid hardcoding URLs directly in your tests. Consider using a configuration file or environment variables to manage different environments.